streamer/src/streamer.rs

```diff
@@ -24,6 +24,13 @@ use {
     thiserror::Error,
 };
 
+// Total stake and nodes => stake map
+#[derive(Default)]
+pub struct StakedNodes {
+    pub total_stake: f64,
```

Comment:
nit: why is this an `f64`? Seems like a `u64` would make more sense

Reply:
I guess this is mostly due to its float arithmetic? Like?

                            VarInt::from_u64(
                                ((stake as f64 / total_stake as f64)
                                    * QUIC_TOTAL_STAKED_CONCURRENT_STREAMS)
                                    as u64,
                            )
                            .unwrap(),

Reply:
should just flip the ops and drop the floats.  floats are the devil